packages/components/src/components/Flyout/__tests__/Flyout.ct.tsx

```diff
@@ -0,0 +1,469 @@
+    const tooltipContent = page.getByTestId(FLYOUT_CONTENT_TEST_ID);
+    await expect(component).toBeVisible();
+    await expect(tooltipTrigger).toBeVisible();
+    await expect(component).toContainText(FLYOUT_TRIGGER_TEXT);
+    await expect(tooltipContent).not.toBeVisible();
+    await tooltipTrigger.click();
+    await expect(tooltipContent).toBeVisible();
+    await page.click('body');
+    await expect(tooltipContent).not.toBeVisible();
+});
+
+test('should close on cross icon click', async ({ mount, page }) => {
+    const component = await mount(
+        <Flyout.Root>
+            <Flyout.Trigger>
+                <Button>{FLYOUT_TRIGGER_TEXT}</Button>
+            </Flyout.Trigger>
+            <Flyout.Content>
+                <Flyout.Header showCloseButton>{FLYOUT_HEADER_TEXT}</Flyout.Header>
+                <Flyout.Body>{FLYOUT_BODY_TEXT}</Flyout.Body>
+                <Flyout.Footer>
+                    <div className="tw-flex tw-justify-end tw-gap-2">
+                        <Button>{FLYOUT_FOOTER_TEXT}</Button>
+                    </div>
+                </Flyout.Footer>
+            </Flyout.Content>
+        </Flyout.Root>,
+    );
+    const tooltipTrigger = page.getByTestId(FLYOUT_TRIGGER_TEST_ID);
+    const tooltipContent = page.getByTestId(FLYOUT_CONTENT_TEST_ID);
+    await expect(component).toBeVisible();
+    await expect(tooltipTrigger).toBeVisible();
+    await expect(component).toContainText(FLYOUT_TRIGGER_TEXT);
+    await expect(tooltipContent).not.toBeVisible();
+    await tooltipTrigger.click();
+    await expect(tooltipContent).toBeVisible();
+    await page.getByTestId('fondue-flyout-header-close').click();
+    await expect(tooltipContent).not.toBeVisible();
+});
+
+test('should not render close icon when prop is not passed', async ({ mount, page }) => {
+    const component = await mount(
+        <Flyout.Root>
+            <Flyout.Trigger>
+                <Button>{FLYOUT_TRIGGER_TEXT}</Button>
+            </Flyout.Trigger>
+            <Flyout.Content>
+                <Flyout.Header>{FLYOUT_HEADER_TEXT}</Flyout.Header>
+                <Flyout.Body>{FLYOUT_BODY_TEXT}</Flyout.Body>
+                <Flyout.Footer>
+                    <div className="tw-flex tw-justify-end tw-gap-2">
+                        <Button>{FLYOUT_FOOTER_TEXT}</Button>
+                    </div>
+                </Flyout.Footer>
+            </Flyout.Content>
+        </Flyout.Root>,
+    );
+    const tooltipTrigger = page.getByTestId(FLYOUT_TRIGGER_TEST_ID);
+    const tooltipContent = page.getByTestId(FLYOUT_CONTENT_TEST_ID);
+    await expect(component).toBeVisible();
+    await expect(tooltipTrigger).toBeVisible();
+    await expect(component).toContainText(FLYOUT_TRIGGER_TEXT);
+    await expect(tooltipContent).not.toBeVisible();
+    await tooltipTrigger.click();
+    await expect(tooltipContent).toBeVisible();
+    await expect(page.getByTestId('fondue-icons-cross')).not.toBeVisible();
```

Comment:
Don't forget to update the usages of the test id to use the new one

Reply:
adjusted